shipment_advice/models/shipment_advice.py

```diff
@@ -0,0 +1,385 @@
+            "draft": [("readonly", False)],
+            "confirmed": [("readonly", False)],
+            "in_progress": [("readonly", False)],
+        },
+        domain=[("package_level_id", "=", False)],
+        readonly=True,
+    )
+    loaded_move_lines_without_package_count = fields.Integer(compute="_compute_count")
+    loaded_picking_ids = fields.One2many(
+        comodel_name="stock.picking",
+        compute="_compute_picking_ids",
+        string="Loaded transfers",
+    )
+    loaded_pickings_count = fields.Integer(compute="_compute_count")
+    loaded_package_ids = fields.One2many(
+        comodel_name="stock.quant.package",
+        compute="_compute_package_ids",
+        string="Packages",
+    )
+    loaded_package_level_ids = fields.One2many(
+        comodel_name="stock.package_level",
+        compute="_compute_package_ids",
+        string="Package Levels",
+    )
+    loaded_packages_count = fields.Integer(compute="_compute_count")
+    carrier_ids = fields.Many2many(
+        comodel_name="delivery.carrier",
+        string="Related shipping methods",
+        compute="_compute_carrier_ids",
+        help=(
+            "Concerned shipping method  for this shipment advice. It can be "
+            "used to determine what are the eligible deliveries to load in "
+            "the shipment when you don't have planned content "
+            "(e.g. through the shopfloor application)."
+        ),
+    )
+
+    _sql_constraints = [
+        (
+            "name_uniq",
+            "unique(name, company_id)",
+            "Reference must be unique per company!",
+        ),
+    ]
+
+    @api.depends("loaded_package_ids")
+    def _compute_total_load(self):
+        for shipment in self:
+            packages = self.loaded_move_line_ids.result_package_id
+            shipment.total_load = sum(packages.mapped("shipping_weight"))
+
+    @api.depends("planned_move_ids", "loaded_move_line_ids")
+    def _compute_picking_ids(self):
+        for shipment in self:
+            shipment.planned_picking_ids = shipment.planned_move_ids.picking_id
+            shipment.loaded_picking_ids = shipment.loaded_move_line_ids.picking_id
+
+    @api.depends("loaded_move_line_ids.package_level_id.package_id",)
+    def _compute_package_ids(self):
+        for shipment in self:
+            shipment.loaded_package_level_ids = (
+                shipment.loaded_move_line_ids.package_level_id
+            )
+            package_ids = set()
+            for line in shipment.loaded_move_line_ids:
+                if line.package_level_id:
+                    package_ids.add(line.package_level_id.package_id.id)
+            shipment.loaded_package_ids = self.env["stock.quant.package"].browse(
+                package_ids
+            )
+
+    @api.depends("planned_picking_ids", "planned_move_ids")
+    def _compute_count(self):
+        for shipment in self:
+            shipment.planned_pickings_count = len(self.planned_picking_ids)
```

Comment:
```suggestion
            shipment.planned_pickings_count = len(shipment.planned_picking_ids)
```
Same for the following lines